index.html

```diff
@@ -7,7 +7,7 @@
     <title>Hello Microverse</title>
 </head>
 <body>
-    <h1>Hello Microverse</h1>
+    <h1 class="header">Hello Microverse</h1>
     <h2>Easy to understand commit messages</h2>
     <h3>Easy to understand descriptions of pull requests</h3>
```

Comment:
- [ ] In my opinion, It would be better if you desplayed only "Hello world!" as recommended by the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) :recycle: